errors.js

```diff
@@ -0,0 +1,9 @@
+exports.serverErrorHandler = (err, req, res, next) => {
+    console.log(err);
+    res.status(500).send({msg: 'Sorry, I wrote some bad code...'});
+    next(err);
```

Comment:
No need to invoke next with your 500 as this will be the last part of your error handler